src/Microsoft.Health.Fhir.SqlServer/Features/Search/SqlServerSearchService.cs

```diff
@@ -926,16 +926,21 @@ protected async override Task<SearchResult> SearchForReindexInternalAsync(Search
             }
 
             var queryHints = searchOptions.QueryHints;
-            long startId = long.Parse(queryHints.First(_ => _.Param == KnownQueryParameterNames.StartSurrogateId).Value);
-            long endId = long.Parse(queryHints.First(_ => _.Param == KnownQueryParameterNames.EndSurrogateId).Value);
+            long globalStartId = long.Parse(queryHints.First(_ => _.Param == KnownQueryParameterNames.StartSurrogateId).Value);
+            long globalEndId = long.Parse(queryHints.First(_ => _.Param == KnownQueryParameterNames.EndSurrogateId).Value);
```

Comment:
:see_no_evil: nit: using `_` has a convention of being the same name as [discard](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/discards) variables

Reply:
I only changed the variable names here for clarity but am happy to fix this in another PR if needed. 😄

Reply:
Actually fixed this since pipelines need to run again. 👍🏽